integrations-and-supported-tools/pytorch/scripts/Neptune_Pytorch_Support.py

```diff
@@ -1,89 +1,85 @@
-from numpy.random import permutation
-import torch
-import torch.nn as nn
-import torch.optim as optim
-import torchvision
-from torchvision import datasets, models, transforms
-import neptune.new as neptune
-import numpy as np
-
-# Step 1: Initialize Neptune and create new Neptune Run
-run = neptune.init(project="common/pytorch-integration", tags='Basic script', api_token="ANONYMOUS", source_files=["*.py"])
-
-# Experiment Config
-data_dir = "data/CIFAR10"
-compressed_ds = "./data/CIFAR10/cifar-10-python.tar.gz"
-data_tfms = {
-        "train": transforms.Compose([
-            transforms.RandomHorizontalFlip(),
-            transforms.ToTensor(),
-            transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-        ])
+def main():
```

Comment:
Let's not do it like this but rather let tests fail and devs fix it @SiddhantSadangi 

Our users shouldn't have to do it this way or if the have to let's now it for sure and have product decide.